app/main/helpers/suppliers.py

```diff
@@ -86,6 +86,18 @@ def is_g12_recovery_supplier(supplier_id: Union[str, int]) -> bool:
     return int(supplier_id) in supplier_ids
 
 
+def get_g12_recovery_draft_ids() -> Set[int]:
+    draft_ids_string = current_app.config.get('DM_G12_RECOVERY_DRAFT_IDS') or ''
+
+    try:
+        return {int(s) for s in draft_ids_string.split(sep=',')}
+    except AttributeError as e:
+        current_app.logger.error("DM_G12_RECOVERY_DRAFT_IDS not a string", extra={'error': str(e)})
+    except ValueError as e:
+        current_app.logger.error("DM_G12_RECOVERY_DRAFT_IDS not a list of draft IDs", extra={'error': str(e)})
+    return set()
```

Comment:
It might be better to do this once when initialising the config object rather than every time we need the list?

Reply:
I agree that that would be preferable. Do you know of a way to do that without conflicting with the magic that loads config values from environment variables? Is there any precedence for this?

If there's a known way of doing this, I think it would be worth switching. Otherwise, I don't think it would be worth the trouble for something that only gets called once per page, and where the code will be deleted in a month's time.

Reply:
I think I've worked out a way of doing it - I can load the list inside `init_app`.

Since this applies to both the list of draft IDs and supplier IDs, I'd like to suggest merging this PR as-is. I'll sort out loading this only once in a follow-up PR.